packages/design-tokens/src/utils/paddingDefaultsHelper.ts

```diff
@@ -66,12 +66,14 @@ export const paddingDefaultsHelper = (
     pt: py ? undefined : pt,
     px,
     py,
-  }
+  } as const
+
+  type responseKey = keyof typeof response
```

Comment:
Are we strict about capitalizing type alias?
```suggestion
  type ResponseKey = keyof typeof response
```

Reply:
No but we probably should be :)